Traffic-Based App Rewards: Add Scan API docs for app activity records (#5513)
hyperledger-labs/splice: docs/src/app_dev/scan_api/scan_bulk_data_api.rst

   * **confirming_parties**: The parties responsible for confirming the validity of the transaction view, along with their quorum threshold.
   * **sub_views**: Other views that the current one depends on, referred to by their ``view_id`` fields.

Traffic summary
^^^^^^^^^^^^^^^

.. warning::

   This field is experimental while CIP-0104 traffic-based app rewards are in preview.

>    It will be stabilised once CIP-0104 exits the preview phase and is fully enabled on mainnet.

Reviewer: add an .rst comment with a `TODO` linking to the issue to remove it.

Author: Fixed

>    It will be stabilised once CIP-0104 exits the preview phase and is fully enabled on mainnet.

Reviewer: ```suggestion
   Until CIP-0104 exits the preview phase and is fully enabled on MainNet,
   this field may not always be served and its format can still change.
```

here and in the other comment

> Each event may include a ``traffic_summary`` field containing traffic cost data

Reviewer: ```suggestion
Each event includes a ``traffic_summary`` field containing traffic cost data
```

from the sequencer for the confirmation request corresponding to the event.

The ``traffic_summary`` object contains:

* **total_traffic_cost** : Total traffic cost in bytes of the confirmation request, paid by the
  validator node that submitted it.
* **envelope_traffic_summaries** : An array of per-envelope traffic data. Each entry contains:

   * **traffic_cost**: Traffic cost in bytes for this envelope.
   * **view_ids**: View IDs from the verdict contained in this envelope. These correspond to the
     ``view_id`` fields in the ``transaction_views`` array.

App activity records
^^^^^^^^^^^^^^^^^^^^

.. warning::

   This field is experimental while CIP-0104 traffic-based app rewards are in preview.
   It will be stabilised once CIP-0104 exits the preview phase and is fully enabled on mainnet.

Each event may include an ``app_activity_records`` field containing per-app-provider
activity weights computed from traffic summaries as per
`CIP-0104 <https://github.com/canton-foundation/cips/blob/main/cip-0104/cip-0104.md>`_.
This is present when a traffic summary is available for the event.

The ``app_activity_records`` object contains:

> * **round_number** : The mining round number assigned to the activity records.

Reviewer: ```suggestion
* **round_number** : The minting round number assigned to the activity records.
```

Reviewer: The official language is "minting". The code is off.

https://www.canton.network/hubfs/Canton%20Network%20Files/Documents%20(whitepapers,%20etc...)/Canton%20Coin_%20A%20Canton-Network-native%20payment%20application.pdf
